configure.ac

```diff
@@ -413,7 +413,7 @@ case $host in
        AC_MSG_ERROR("windres not found")
      fi
 
-     CPPFLAGS="$CPPFLAGS -D_MT -DWIN32 -D_WINDOWS -DBOOST_THREAD_USE_LIB"
+     CPPFLAGS="$CPPFLAGS -D_MT -DWIN32 -D_WINDOWS -DBOOST_THREAD_USE_LIB -D_WIN32_WINNT=0x0601"
      LEVELDB_TARGET_FLAGS="-DOS_WINDOWS"
      if test "x$CXXFLAGS_overridden" = "xno"; then
        CXXFLAGS="$CXXFLAGS -w"
```

Comment:
https://github.com/bitcoin/bitcoin/pull/18928

Two birds with one stone?

Reply:
Looks like the change is already there.

Reply:
the change gets rid of lines 417-420 altogether

Reply:
We will do this in Janice since the PR is already merged.

Reply:
I take that back. I will do this in the other PR I am doing to fix up the NSI installer.